pygaggle/model/evaluate.py

```diff
@@ -75,7 +75,7 @@ def accumulate(self, scores: List[float], gold: RelevanceExample):
         score_rels = self.truncated_rels(scores)
         score_rels[score_rels != 0] = 1
         gold_rels = np.array(gold.labels, dtype=int)
-        score = recall_score(gold_rels, score_rels, zero_division=1)
+        score = recall_score(gold_rels, score_rels, zero_division=0)
```

Comment:
Yeah, that makes sense. @daemon, before we merge this, I just want to double-check with you if there was a reason for  `zero_division=1`

Reply:
No objection.